packages/browser/package.json

```diff
@@ -20,7 +23,9 @@
     "@sentry/replay": "7.36.0",
     "@sentry/types": "7.36.0",
     "@sentry/utils": "7.36.0",
-    "tslib": "^1.9.3"
+    "tslib": "^1.9.3",
+    "yargs": "^17.6.2",
+    "uuid": "^9.0.0"
```

Comment:
let's try and avoid adding these deps if we can!

Reply:
How?

Reply:
`uuid` we can just generate ourselves, we don't need strict `uuid4` (copy paste the implementation in Sentry you have to)

`yargs` is more painful, but just parse the args ourselves. We don't have a super complicated CLI atm, so no need to worry about ordering concerns or similar for now.

Reply:
As discussed: Since this is only a poc I removed the CLI